Sources/SFSafeSymbols/Symbols/LocalizableSFSymbol.swift

```diff
@@ -0,0 +1,203 @@
+
+@available(iOS 13.0, macOS 11.0, tvOS 13.0, watchOS 6.0, *)
+public protocol SymbolLocalization {
+    init(source: SFSymbol)
+}
+
+// TODO: replace the following with Variadic Generics once available
+// Since the code is currently only an interim solution, we don't auto-generate it.
+// Beware: if more localizations are added to SFSymbols before variadic generics are introduced to Swift, you may have to create the missing specialized classes.
+
+@available(iOS 13.0, macOS 11.0, tvOS 13.0, watchOS 6.0, *)
+public /* abstract */ class LocalizableSFSymbol: SFSymbol {
```

Comment:
What do you think: Maybe we can name this `SFSymbolWithLocalizationInformation` instead of `LocalizableSFSymbol`...

`LocalizableSFSymbol` is somehow misleading. I thought about this when reading your comment in #100: What will the user think when they are presented a `Set<LocalizableSymbol`...

Reply:
You're right, we don't need the distinction between `SFSymbol` and `LocalizableSFSymbol`. I'll change it.